src/page/registration/registration.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {Props} from 'react';
+import {View, Text, TextInput, TouchableHighlight} from 'react-native';
+import {styles} from './registrationCss';
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+import {faArrowLeft} from '@fortawesome/free-solid-svg-icons';
+import Logo from '../../resource/images/logo.svg';
+
+const Registration: React.FC = (Props: Props) => {
+  return (
+    <View style={styles.container}>
```

Comment:
This view is not corresponding to the design. Please add gradient for background, reduce bottom offset and add 
[KeyboardAvoidingView](https://reactnative.dev/docs/keyboardavoidingview) to adjust inputs to keyboard